cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -4366,3 +4366,20 @@ ir.cpp:
 #  987|     v0_21(void)             = ReturnValue              : r0_20, mu0_2
 #  987|     v0_22(void)             = UnmodeledUse             : mu*
 #  987|     v0_23(void)             = ExitFunction             : 
+
+#  991| int AsmStmt(int)
+#  991|   Block 0
+#  991|     v0_0(void)        = EnterFunction            : 
+#  991|     mu0_1(unknown)    = AliasedDefinition        : 
+#  991|     mu0_2(unknown)    = UnmodeledDefinition      : 
+#  991|     r0_3(glval<int>)  = VariableAddress[x]       : 
+#  991|     mu0_4(int)        = InitializeParameter[x]   : r0_3
+#  992|     mu0_5(unknown)    = InlineAsm                : mu0_2
+#  993|     r0_6(glval<int>)  = VariableAddress[#return] : 
+#  993|     r0_7(glval<int>)  = VariableAddress[x]       : 
+#  993|     r0_8(int)         = Load                     : r0_7, mu0_2
```

Comment:
Can inline assembly actually change the value of a non-escaping local variable (`x` in this case)? I don't see that on https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html.

Reply:
That's unmodeled because this is raw IR, not because we can't model it after SSA analysis.

Reply:
Oh, I misread.